### ISSUE TITLE
In der Bilanz für genutzte Zwischenfrüchte fehlen der P und K Entzug

### ISSUE THREAD

**Issue by simonkriegnerschramml** [w3geo/bml-fast#147]
Hallo Arnold,

Im Zuge der Bilanzdarstellung bin ich drauf gekommen, dass bei genutzten Zwischenfrüchten kein P Entzug und kein K Entzug abgebildet wird. Auch in der Berechnung fehlen diese. Liegt das möglicherweise an der fehlenden Ernteverbuchung (Die brauchen wir ja vereinfacht nicht hier).

LG
Simon

**Comment by arnold-pichler** [comment]
Stimmt. Das muss analog zum N-Entzug sein. Schau ich mir noch an. Danke.

**Comment by arnold-pichler** [comment]
Wurde gefixed, bitte kontrollieren.
